### ISSUE TITLE
Invalid IEx autocompletion for `al/<tab>`

### ISSUE THREAD

**Issue by kelvinst** [elixir-lang/elixir#10910]
### Environment

* Elixir & Erlang/OTP versions (elixir --version): 

```
Erlang/OTP 22 [erts-10.7.1] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe]

Elixir 1.11.4 (compiled with Erlang/OTP 21)
```

* Operating system: Mac OS

### Current behavior

When you type `al/` and hit tab on IEx, it autocompletes to `al/ias`

### Expected behavior

Not to complete that at all, I think the autocompletion for expressions ending with `/` are useful for the capture operator `&` and nothing else.

Got this bug while testing a refactor I'm doing for https://github.com/elixir-lang/elixir/pull/10895, so maybe I can fix the issue in there, just registering so we don't lose track of that.

**Comment by josevalim** [comment]
Thanks for the report! Let’s fix this separately! PRs also welcome!

**Comment by josevalim** [comment]
This will be fixed as part of #10915. Thank you!
